### DISCUSSION TITLE
Beep codes

### DISCUSSION THREAD

**Discussion by Lyawon** [640-KB/GLaBIOS#48] [Tech Support]
Hello.
I've just soldered by myself the NuXT v1.2 and had trouble firing it up. I've got beeps on and off with Sergey's BIOS which means "general failure" so I've used GLaBIOS to troubleshoot it more precisely. With GLaBIOS I hear 3 long and 1 short beeps. Is it the same as 1 short and 3 long beeps (Base RAM (0-16KB) read/write error)?

I have included the short video here for you to refer to. 
https://github.com/640-KB/GLaBIOS/assets/173090516/00e89c8a-39d3-4e2e-b690-b58d6df052ea

**Comment by 640-KB** [comment]
Yeah, that's exactly what I hear as well - 3 long/1 short is memory test failure.  In this case the memory is failing from pretty much the start so likely it's going to just be RAM in general isn't working.

Do you have a known working POST code card?  I can get you a build that will output the memory segment/address of where it's failing.... might get lucky and be able to trace out a specific line that's not working.

Have you tried [Ruud's diagnostic ROM](https://www.minuszerodegrees.net/ruuds_diagnostic_rom/Ruuds%20Diagnostic%20ROM.htm) or [Supersoft/Landmark](https://www.minuszerodegrees.net/supersoft_landmark/Supersoft%20Landmark%20ROM.htm)?  They may get you some more specific information, though you might need a CGA in order for them to work fully.

**Reply by Lyawon** [reply]
Thanks a lot for confirming that it is a memory failure. Also, links are useful. 
Unfortunately, I do not have POSTcard or CGA/MDA/EGA. Also, my IC programmer is not with me for now.
I'm going to start the diagnostic with a multimeter =) I'll get back to you if I can't find anything.

UPD: To inform you, It was a short circuit of DATA3 on the xt-bus to the ground.
